app/employee/dashboard/page.tsx

```
@@ -235,6 +306,57 @@ export default function EmployeeDashboardPage() {
+            <div className="space-y-2">
+              <label className="text-sm font-medium">
+                Date
+                <div className="mt-2 rounded-md border p-3">
+                  <DateCalendar
+                    mode="single"
+                    selected={selectedDate}
+                    onSelect={setSelectedDate}
+                    className="w-full"
+                  />
+                </div>
+              </label>
```

Biome (2.4.6)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)